app/js/components/service_form.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+let serviceForm = new function($) {
```

Comment:
Maybe rebase onto develop and reap the power of TypeScript. This might improve testability too?

Reply:
I've unleashed the power of Typescript...